src/Graphics/tests/Graphics.Tests/FontUnitTests.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Xunit;
+
+namespace Microsoft.Maui.Graphics.Tests
+{
+	public class FontUnitTests
```

Comment:
Add a test to assert that `Font.Default` _does not equal_ `Font.DefaultBold`.

Reply:
Sure, added. Could you review it again?. Thnx!